models/profile.rb

```diff
@@ -3,4 +3,7 @@
 class Profile < ApplicationRecord
   belongs_to :user
   validates :first_name, presence: true, uniqueness: true
+
+  def set_age
```

Comment:
empty method. please remove

Reply:
updating code today actually i was experimenting to get user details there